recipes/tbb/all/conanfile.py

```diff
@@ -103,7 +98,7 @@ def add_flag(name, value):
                 add_flag('CFLAGS', '-mrtm')
                 add_flag('CXXFLAGS', '-mrtm')
 
-            targets = self.get_targets()
+            targets = ["tbb", "tbbmalloc", "tbbproxy"]
```

Comment:
tbbmalloc and tbbproxy are optional, should we force both as targets?

Reply:
we are forcing the build always, but generating only one package. See `package_id()`